src/views/certificate-signed.njk

```diff
@@ -0,0 +1,36 @@
+{% extends 'layout.njk' %}
+
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block pageTitle %}
+  {{ serviceName }}
```

Comment:
Pretty sure that this should match the heading of the page. Just did a quick scan through our repo and they seem to be really inconsistent. Can you update these to match the heading of the respective pages?

Reply:
Will do!